### PR TITLE
New package: novaboot, perl-Expect (dependency for novaboot)

### PR DESCRIPTION
**Pull request by ametisf**
Hi, this is my first public package. Hope everything is ok, I'm not super sure about how to use depends/makedepends with perl.

### CODE REVIEW

**Comment by Hoshpak** [comment]
Why did you close the PR? I was just about to review it.

**Comment by ametisf** [comment]
I just noticed a missing dependency, fixing it now.

**Comment by Hoshpak** [comment]
Alright but you don't need to close the PR to do that. Just amend your commits and force-push to the remote branch to update the PR. Also please put each new package into its own commit, so that should be two separate commits for this PR.

File: srcpkgs/novaboot/template [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Template file for 'novaboot'
+pkgname=novaboot
+version=20180323
+revision=1
+noarch=yes
+makedepends="perl"
+depends="perl-Expect"
+short_desc="Tool that automates booting of operating systems on target hardware or in qemu."
+maintainer="adam <ametisf@gmail.com>"
+license="GPL-2"
```
[the remark this answers is not in the document]
Comment (ametisf) [reply srcpkgs/novaboot/template]: Oh, I didn't know about this, should've read the manual more carefuly. Thank you.

File: srcpkgs/novaboot/template [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+homepage="https://github.com/wentasah/novaboot"
+distfiles="https://github.com/wentasah/novaboot/archive/${version}.tar.gz"
+checksum=f1a8781d9bb470d631014063986fb89334747bb718e3acef33d24a35f4359548
+
+do_build() {
```
[the remark this answers is not in the document]
Comment (ametisf) [reply srcpkgs/novaboot/template]: Unfortunately no, novaboot is not a complete perl-module and doesn't contain all the necessary files to build using perl-module build style.

**Comment by ametisf** [comment]
I think I've fixed everything now. Do you know if there is a way to make github reopen a pull request after I force-pushed into the original branch, or should I make new pull request? Thank you very much for your time.

**Comment by ametisf** [comment]
Created new pull request #4559

**Pull request closed without merging**